### ISSUE TITLE
tintColor style property not having any effect

### ISSUE THREAD

**Issue by bbqsrc** [ptmt/react-native-macos#156]
The UIExplorer test app shows the issue:

<img width="519" alt="screen shot 2017-04-15 at 15 27 20" src="https://cloud.githubusercontent.com/assets/279099/25061028/16efabee-21f0-11e7-9e52-ef5b9c1e6a78.png">

As you can see, they're all the same color when they should not be. 😉

**Comment by ptmt** [comment]
Yes, NSImage has no `tintColor` as UIImage on iOS does. So currently it's like a placeholder. Although,  I can implement it by applying image transformations.

**Comment by bbqsrc** [comment]
Yeah, my current experimentation with that has got me to here:

```
extension NSImage {
    func tint(color: NSColor) {
        lockFocus()
        color.set()
        NSRectFillUsingOperation(NSMakeRect(0, 0, self.size.width, self.size.height), .sourceAtop)
        unlockFocus()
        isTemplate = false
    }
}
```
